about.html

```diff
@@ -0,0 +1,149 @@
+                    <a href="" class="gray">
+                        <i class="fa-brands fa-twitter">Twitter</i>
+                    </a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">English</a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">My page</a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <div class="overlay">
+            <nav class="main-menu">
+                <a href="index.html" class="logo">
+                    <img src="./images/selDev-logo.png" alt="logo">
+                </a>
+                <ul class="nav">
+                    <li class="nav-link">
+                        <a href="" class="dark">About</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Services</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Coaches</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Resources</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Get In Touch</a>
+                    </li>
+                </ul>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="container about">
+            <div class="about-title">
+                <h1 class="orange">CREATE COMMON <br>GLOBAL SUMMIT 2015</h1>
+            </div>
+            <div  class="about-text">
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
+                </p>
+            </div>
+            <div class="about-sub-text">
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            </p>
+                <p>@2022, SelfDev. All rights reserved</p>
+            </div>
+        </section>
```

Comment:
- [ ] Good job designing this section 😄 Kindly add a background image so that it may closely follow the original design and also improve the design of your page 💯 
- Your design:
![Annotation 2022-05-26 085028](https://user-images.githubusercontent.com/85113567/170448818-04df00cd-04da-4e06-a368-91af8c997330.jpg)
- Required design:
![Annotation 2022-05-26 085045](https://user-images.githubusercontent.com/85113567/170448863-806e8c67-819a-4ae5-91b3-a2376b85717a.jpg)